Makefile

```diff
@@ -25,10 +25,10 @@ profile:
 	@cp vimrc ~/.vimrc
 
 k8s:
-	curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.16.5/bin/linux/amd64/kubectl
+	curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/linux/amd64/kubectl
```

Comment:
let's use the latest patch release for the 1.20 line: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#v1209